### PR TITLE
updted skill_test data dictionaty with new entry.

### PR DESCRIPTION
**Pull request by mihirdcoder**
## Description
Updated skill_test.py file as first exercise of the Padawan program.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/syft/tests/trials/skill_test.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def get_padawans(cohort: str) -> Dict[str, str]:
             "rodrigo": "PASSED",
             "kanak": "PASSED",
             "theresa": "PASSED",
+            "mihir":"PASSED"
```
Comment (rasswanth-s) [inline packages/syft/tests/trials/skill_test.py:19]: NIT: could you also include `,` at the end.

Comment (mihirdcoder) [reply packages/syft/tests/trials/skill_test.py]: Added, thanks!

**Comment by rasswanth-s** [comment]
@mihirdcoder  could you revert the protobuf changes in this PR, `path: packages/syft/src/syft/proto/*` , I think you must have formatted it with a new version, we currently use 3.19.4

**Review by rasswanth-s — APPROVED** [review]
Great work @mihirdcoder  on your first padawan PR 🎸

**Pull request merged**